src/bridger/Bridger.sol

```diff
@@ -140,25 +144,27 @@ contract Bridger is
 
     /**
      * @dev Deposit the specified amount of tokens in to the Kinto L2
-     * @param _kintoWallet Kinto Wallet Address on L2 where tokens will be deposited
      * @param _signatureData Struct with all the required information to deposit via signature
-     * @param _swapData Struct with all the required information to swap the tokens
      * @param _permitSignature Signature to be recovered to allow the spender to spend the tokens
      */
-    function depositBySig(
-        address _kintoWallet,
-        IBridger.SignatureData calldata _signatureData,
-        IBridger.SwapData calldata _swapData,
-        bytes calldata _permitSignature
-    ) external payable override whenNotPaused onlySignerVerified(_signatureData) onlyPrivileged {
+    function depositBySig(IBridger.SignatureData calldata _signatureData, bytes calldata _permitSignature)
+        external
+        payable
+        override
+        whenNotPaused
+        nonReentrant
+        onlyPrivileged
+        onlySignerVerified(_signatureData)
+    {
         _isFinalAssetAllowed(_signatureData.finalAsset);
-        // Only allows assets for now which do not require swap.
         if (_signatureData.inputAsset != _signatureData.finalAsset && !allowedAssets[_signatureData.inputAsset]) {
             // checks for USDe special case
-            if (_signatureData.inputAsset != USDe && _signatureData.finalAsset != sUSDe) {
+            if (_signatureData.inputAsset != USDe || _signatureData.finalAsset != sUSDe) {
```

Comment:
```suggestion
            if (_signatureData.inputAsset == USDe && _signatureData.finalAsset != sUSDe) {
```
Should it be this? Probably need a test case for this.

Reply:
It has liquidity in matcha so we can allow other final assets